### PR TITLE
S2U-39 rWiki conversion scripts update

### PR DESCRIPTION
**Pull request by JuanDavid102**
This is linked with https://github.com/sakaiproject/sakai/pull/11704

### CODE REVIEW

File: docs/conversion/s2u_features/S2U-39-oracle.sql [hunk 1]
```diff
@@ -1,3 +1,6 @@
 -- S2U-39 --
-ALTER TABLE rwikiobject ADD page_groups CLOB DEFAULT '';
+Create table rwikipagegroups (
+    rwikiobjectid VARCHAR(36) NOT NULL,
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-39-oracle.sql:3]: VARCHAR2(36)

File: docs/conversion/s2u_features/S2U-39-oracle.sql [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+    groupid VARCHAR(99) NOT NULL
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-39-oracle.sql:4]: VARCHAR2(99)

File: docs/conversion/s2u_features/S2U-39-oracle.sql (hunk 1 above)
Comment (mpellicer) [reply docs/conversion/s2u_features/S2U-39-oracle.sql]: I've been faster than you Jesus

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
WDYT @jesusmmp ?

File: docs/conversion/s2u_features/S2U-39-mysql.sql [hunk 3]
```diff
@@ -1,3 +1,6 @@
 -- S2U-39 --
-ALTER TABLE rwikiobject ADD page_groups LONGTEXT DEFAULT '';
+CREATE TABLE rwikipagegroups (
+    rwikiobjectid VARCHAR(36) NOT NULL,
+    groupid VARCHAR(99) NOT NULL
+)
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-39-mysql.sql:5]: );

File: docs/conversion/s2u_features/S2U-39-oracle.sql [hunk 4]
```diff
@@ -1,3 +1,6 @@
 -- S2U-39 --
-ALTER TABLE rwikiobject ADD page_groups CLOB DEFAULT '';
+CREATE TABLE rwikipagegroups (
+    rwikiobjectid VARCHAR2(36) NOT NULL,
+    groupid VARCHAR2(99) NOT NULL
+)
```
Comment (jesusmmp) [inline docs/conversion/s2u_features/S2U-39-oracle.sql:5]: );

**Review by jesusmmp — APPROVED** [review]

**Pull request merged**